docs/_includes/markup/feature-many-icons.njk

```diff
@@ -1,17 +1,17 @@
 <div class="feature-many-icons">
-    <div class="container-fluid">
-        <div class="row align-items-center">
-            <div class="col-md-6">
-                <h2 class="feature-big-text"><span class="fas fa-lock mb-2"></span><br />Big Text.</h2>
-            </div>
-            <div class="col-md-6 text-white">
-                <div class="mb-2 text-white text-lg-left">
-                    <span class="fas fa-shield-alt"></span>
-                    <span class="fas fa-user-lock mx-4"></span>
-                    <span class="fas fa-id-card"></span>
-                </div>
-                <p class="feature-small-text">This is smaller text which explains this block.</p>
-            </div>
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col-md-6">
+        <h2 class="feature-big-text"><span class="fas fa-lock mb-2"></span><br />Big Text.</h2>
+      </div>
+      <div class="col-md-6 text-white">
+        <div class="mb-2 text-white text-lg-left">
+          <span class="fas fa-shield-alt"></span>
+          <span class="fas fa-user-lock mx-4"></span>
+          <span class="fas fa-id-card"></span>
         </div>
+        <p class="feature-small-text">This is smaller text which explains this block.</p>
+      </div>
     </div>
-</div>
+  </div>
+</div>
```

Comment:
needs a newline cause of Github indicator.

Reply:
@frogpad7 i added the blank line